services/cache/lru/lru.go

```diff
@@ -8,16 +8,18 @@
 // LRU implementation in groupcache:
 // https://github.com/golang/groupcache/tree/master/lru
 
-package utils
+package lru
 
 import (
 	"container/list"
 	"sync"
 	"time"
+
+	"github.com/mattermost/mattermost-server/v5/services/cache"
 )
 
-// Cache is a thread-safe fixed size LRU cache.
-type Cache struct {
+// LruCache is a thread-safe fixed size LRU cache.
+type LruCache struct {
```

Comment:
Same as above; this should be `Cache`. The package name is sufficient to identify that it's an LRU cache.

Reply:
Done!